online_meetings/README.md

```diff
@@ -0,0 +1,2 @@
+# zoom_report_script
+script for report Zoom usage
```

Comment:
We don't need this file, right?

Reply:
Yeah, I don't think it'll be useful to have readme's in subdirectories, I'll remove